### PR TITLE
Fetch chunks method added to client

### PR DESCRIPTION
**Pull request by AleF83**

### CODE REVIEW

File: js/react-tweek/package.json [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
   "author": "Soluto",
   "license": "MIT",
   "url": "https://github.com/Soluto/tweek-clients/tree/master/js/react-tweek",
   "repository": "Soluto/tweek-clients",
   "peerDependencies": {
-    "prop-types": "15.x",
-    "react": "15.x",
-    "tweek-client": "^0.1.5",
-    "tweek-local-cache": "0.2.x"
+    "prop-types": "15.6.1",
```
Comment (Yshayy) [inline js/react-tweek/package.json]: prop-types > 15

File: js/react-tweek/package.json [hunk 2]
```diff
@@ -1,33 +1,32 @@
...
+    "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-0": "^6.24.1",
-    "jest": "^21.1.0",
-    "react": "^15.6.1",
-    "react-dom": "^15.6.1",
-    "react-test-renderer": "^15.6.1",
-    "tweek-client": "^0.1.8",
-    "tweek-local-cache": "^0.2.2"
+    "jest": "^22.4.2",
```
Comment (Yshayy) [inline js/react-tweek/package.json]: add react to dev dependencies

File: js/tweek-client/src/TweekClient.ts [hunk 3]
```diff
@@ -77,9 +72,23 @@ export default class TweekClient implements ITweekClient {
     return <Promise<T>>result;
   }
 
+  fetch<T>(path: string, _config?: FetchConfig): Promise<T> {
+    if (_config) {
```
Comment (Yshayy) [inline js/tweek-client/src/TweekClient.ts]: maybe:

File: js/react-tweek/package.json [hunk 4]
```diff
@@ -1,33 +1,32 @@
...
+    "react": ">=15.0.0",
+    "tweek-client": "^0.1.9",
+    "tweek-local-cache": "0.2.4"
   },
   "dependencies": {
-    "humps": "^2.0.1"
+    "humps": "^2.0.1",
+
+    "tweek-client": "^0.1.x",
+    "tweek-local-cache": "^0.2.x"
```
Comment (nataly87s) [inline js/react-tweek/package.json]: these should be optional, because you don't have to use them in the package

**Pull request merged**